### ISSUE TITLE
iron-routes dynamic routes error

### ISSUE THREAD

**Issue by vladblindu** [aldeed/meteor-tabular#36]
I am bothering you again. It's your fault you offered so many great tools :) Thanks, once more.
I'm stuck betwwen the JQuery of the dataTables and the Meteor part as follows:
I made a dynamic route with iron-router:
```
this.route("dashboardCollections", {
	path: "dashboard/:collectionPath",
	layoutTemplate: 'dashboard',
	data:function(){
	     return {
		templName: "tabular",
		contextData:{
 "table":tablesByName[this.params.collectionPath.substr(0,1).toUpperCase()+this.params.collectionPath.substr(1)+"List"],
		    "selector":{},
		    "class":"table table-striped table-bordered table-condensed"
		}
	    }
	}
    });
```
The path being: user, clients, brands which I get in the params and change into UserList, ClientLists and BrandsList which are exactly the names of the tabulaTables which I retrieve with tablesByName.
At this point my template looks exactly as the one in your example. 
```
 {{> Template.dynamic template = templName data=contextData}}

```
What I don't understand, it's how does the dataTables  keep the previous column data in the dataTables settings and doesn't update.  With static route it works perfect. I'm just stubborn and I would really like to fix this dynamic routes style, which ,by the way might interest others as well.

For people requesting editable tabular forms, I put in the TabularTables.Whatever template the following code:
```
{
	    title:"<div><a href='{{pathFor " + _INSERT_TEMPLATE_+"}}' class='btn btn-sm btn-block btn-primary'><i class='glyphicon glyphicon-log-in'></i>&nbsp;Insert</a></div>",
	    tmpl: Meteor.isClient && Template.editables
	}

<template name="editables">
    <a href="_YOUR_UPDATE_TEMPLATE_ROUTE_" class="btn btn-sm btn-primary"><i class="glyphicon glyphicon-pencil"></i></a>
    <a href="_YOUR_DELETE_TEMPLATE_ROUTE_" class="btn btn-sm btn-primary"><i class="glyphicon glyphicon-remove"></i></a>
</template>
```
Maybe it helps someone.I know it's not SO but I couldn't stop.
Thanks again.

**Comment by aldeed** [comment]
1.0 is out and is a complete rewrite. Please retest with 1.0, and if this is still an issue, provide a link to a simple reproduction app or MeteorPad.

**Comment by vladblindu** [comment]
Hi Eric,
I am really sorry for this late replay.
I wanted to thank you for your time. In the mean time I switched to alsagle:reactive-tables, which, though less complex, fit perfectly my needs for now.
I’m sure I will be bothering you agai as I “dive" into autoform which is really brilliant.
I graduated in computers by have been a sound engineer and a post studio owner for more than 18 years now. I do a little dev as a hobby trying to put a few tools together for the management of the studio.
Validatio, formas, error messages have been the reason I never enede up with a  coherent app yet.
Thanks to your effort this is going to be the first release in 20 years!!!

Thanks again, once for every line of code.

Vlad Blindu
vlad.blindu@gmail.com
